iset.mm

```diff
@@ -134498,6 +134507,274 @@ reduced fraction representation (no common factors, denominator
+        wb cfzo crab ssrab2 eqsstri fzo0ssnn0 nn0ssz sstri f1of simprl ffvelrnd
+        sseldi zmulcld ffvelrnda adantrl moddvds syl3anc zmodfzo syl2anc simprr
+        fvmptd3 eqeq12d zcnd subdid breq2d 3bitr4d gcdcomd simp3d eqtrd zsubcld
+        cn0 coprmdvds cq cle clt elfzole1 elfzolt2 modqid syl22anc f1of1 f1fveq
+        zq sylan 3bitr3d sylibd mpan2d sylbid ralrimivva dff13 sylanbrc cen cfn
+        nnzd 1zzd phicld fzfigd f1oeng eulerthlemfi f1finf1o mpbid f1oco f1oeq1
+        ax-mp sylibr ) AUEGUFLZUJMZUUAEUGZBUUACBUHZELZNMZGOMZUIZUKZULZUUAUUAFUL
+        ZADUUAUUBULZUUADUUGULZUUIAUUADEULZUUKJUUADEUMPAUUADUUGUNZUULAUUADUUGUOZ
+        UAUHZUUGLZUBUHZUUGLZQZUAUBUPZUQZUBUUAURUAUUAURUUNAUUADUCUUACUCUHZELZNMZ
+        GOMZUIZUOUUOAUDBCDUUAEUVGGHIUUAVCJUCUDUUAUVFCUDUHZELZNMZGOMUCUDUPZUVEUV
+        JGOUVKUVDUVICNUVCUVHEUSUTVAVDVBUUADUVGUUGUCBUUAUVFUUFUCBUPZUVEUUEGOUVLU
+        VDUUDCNUVCUUCEUSUTVAVDVEVFAUVBUAUBUUAUUAAUUPUUARZUURUUARZVGZVGZUUTGCUUP
+        ELZUURELZVHMZNMZVISZUVAUVPCUVQNMZGOMZCUVRNMZGOMZQZGUWBUWDVHMZVISZUUTUWA
+        UVPGVJRZUWBTRZUWDTRZUWFUWHVPAUWIUVOAUWICTRZCGVKMZUEQZHVLZVMZUVPCUVQAUWL
+        UVOAUWIUWLUWNHVNZVMZUVPDTUVQDVOGVQMZTDUUCGVKMUEQZBUWSVRUWSIUWTBUWSVSVTZ
+        UWSXFTGWAWBWCWCZUVPUUADUUPEAUUADEUOZUVOAUUMUXCJUUADEWDPZVMAUVMUVNWEZWFZ
+        WGZWHZUVPCUVRUWRUVPDTUVRUXBAUVNUVRDRUVMAUUADUUREUXDWIZWJWGZWHZUWBUWDGWK
+        WLUVPUUQUWCUUSUWEUVPBUUPUUFUWCUUAUUGUWSUUGVCZBUAUPZUUEUWBGOUXMUUDUVQCNU
+        UCUUPEUSUTVAUXEUVPUWJUWIUWCUWSRUXHUWPUWBGWMWNWPUVPBUURUUFUWEUUAUUGUWSUX
+        LBUBUPZUUEUWDGOUXNUUDUVRCNUUCUUREUSUTVAAUVMUVNWOUVPUWKUWIUWEUWSRUXKUWPU
+        WDGWMWNWPWQUVPUVTUWGGVIUVPCUVQUVRUVPCUWRWRUVPUVQUXGWRUVPUVRUXJWRWSWTXAU
+        VPUWAGCVKMZUEQZUVAAUXPUVOAUXOUWMUEAGCAGUWOYHZUWQXBAUWIUWLUWNHXCXDVMUVPU
+        WAUXPVGZGUVSVISZUVAUVPGTRZUWLUVSTRUXRUXSUQAUXTUVOUXQVMUWRUVPUVQUVRUXGUX
+        JXEGCUVSXGWLUVPUVQGOMZUVRGOMZQZUVQUVRQZUXSUVAUVPUYAUVQUYBUVRUVPUVQXHRZG
+        XHRZVOUVQXISZUVQGXJSZUYAUVQQUVPUVQTRZUYEUXGUVQXQPAUYFUVOAUXTUYFUXQGXQPV
+        MZUVPUVQUWSRZUYGUVPDUWSUVQUXAUXFWGZUVQVOGXKPUVPUYKUYHUYLUVQVOGXLPUVQGXM
+        XNUVPUVRXHRZUYFVOUVRXISZUVRGXJSZUYBUVRQUVPUVRTRZUYMAUVNUYPUVMAUVNVGZDTU
+        VRUXBUXIWGWJUVRXQPUYJAUVNUYNUVMUYQUVRUWSRZUYNUYQDUWSUVRUXAUXIWGZUVRVOGX
+        KPWJUVPUYRUYOAUVNUYRUVMUYSWJUVRVOGXLPUVRGXMXNWQUVPUWIUYIUYPUYCUXSVPUWPU
+        XGUXJUVQUVRGWKWLAUUADEUNZUVOUYDUVAVPAUUMUYTJUUADEXOPUUADUUPUUREXPXRXSXT
+        YAYBYCUAUBUUADUUGYDYEAUUADYFSZDYGRUUNUULVPAUUAYGRUUMVUAAUEYTAYIAYTAGUWO
+        YJYHYKJUUADYGEYLWNABCDGHIYMUUADUUGYNWNYOUUADUUAUUBUUGYPWNFUUHQUUJUUIVPK
+        UUAUUAFUUHYQYRYS $.
+    $}
+
+    ${
+
+      $d A u v y $.  $d A x y $.  $d A x z $.  $d F u v y $.  $d F x y $.
+      $d F x z $.  $d N u v y $.  $d N x y $.  $d N x z $.  $d S u $.
+      $d ph u y $.  $d ph x y $.  $d ph x z $.
+
+      $( Lemma for ~ eulerth .  The result.  (Contributed by Jim Kingdon,
+         2-Sep-2024.) $)
+      eulerthlemth $p |- ( ph
```

Comment:
this is ~eulerthlem2 in set.mm. Therefore better "(Contributed by Mario Carneiro, 28-Feb-2014.)  (Revised by Jim Kingdon, 2-Sep-2024.)"?

Reply:
Yeah, I guess I should put this on the various lemmas which were extracted from parts of the set.mm eulerthlem2 proof. I'll push that up, maybe later today or tomorrow.

Reply:
Done.